### PR TITLE
Late July pull 2

### PR DESCRIPTION
**Pull request by DandelionSprout**
* Added 3 lists.
* Assigned a bucketload of languages.
* Corrected the homepage for *AdGuard DNS Filter*.
* Removed a whole lot of Repo.or.cz mirrors.
* Removed one assignment from Opera.

### CODE REVIEW

File: data/FilterList.json [hunk 1]
```diff
@@ -1763,7 +1760,6 @@
     "updatedDate": "2009-10-18T00:00:00",
     "viewUrl": "http://git.sourceforge.jp/view?p=ayucat-list/ayucat-list.git;a=blob_plain;f=ayucat-list.txt;hb=b254c74c132832a3ade7b9d42c2ef8d3dd59fdb9",
     "viewUrlMirror1": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AncientLibrary/Ayucat%20Powerful%20List.txt",
```
Comment (collinbarrett) [inline data/FilterList.json:1762]:
```suggestion
    "viewUrlMirror1": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AncientLibrary/Ayucat%20Powerful%20List.txt"
```

Comment (DandelionSprout) [reply data/FilterList.json:1762]: Thanks for pointing it out. Should be fixed now.

File: data/FilterList.json [hunk 2]
```diff
@@ -2325,7 +2321,6 @@
     "updatedDate": "2010-05-18T00:00:00",
     "viewUrl": "http://www.squirrelconspiracy.net/abp/facebook-privacy-list.txt",
     "viewUrlMirror1": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AncientLibrary/Facebook%20Privacy%20List.txt",
```
Comment (collinbarrett) [inline data/FilterList.json:2323]:
```suggestion
    "viewUrlMirror1": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AncientLibrary/Facebook%20Privacy%20List.txt"
```

Comment (DandelionSprout) [reply data/FilterList.json:2323]: This suggestion helped me notice a pattern that I've had to fix after I did some automated text removal in the initial PR. *This* time it should be fixed.

**Comment by collinbarrett** [comment]
A tool like this could be helpful to catch the comma issues. I know the whole process of updating data is still painful...

https://jsonlint.com/

**Pull request merged**